test_utilities/src/aws.rs

```diff
@@ -713,3 +713,60 @@ pub fn non_working_environment(context: &Context) -> Environment {
 
     environment
 }
+
+
+pub fn echo_app_environment(context: &Context) -> Environment {
```

Comment:
can you please add comment or rename the function, we should understand what role it this